update/snapshot.go

```diff
@@ -0,0 +1,19 @@
+package update
+
+import "github.com/spf13/cobra"
+
+// used by CI to inject url for downloading with tatcli update.
+// value of urlUpdate injected at build time
+// full URL update is constructed with architecture var :
+// urlUpdate + "tatcli-" + architecture, tatcli is the binary
+var urlUpdateSnapshot string
```

Comment:
this feature is not exposed by binary build from travis.
see https://github.com/runabove/sail/blob/fixSailUpdate/update/update.go#L18
but I can move 
```
func init() {
	if urlUpdateSnapshot != "" {
		Cmd.AddCommand(cmdUpdateSnapshot)
	}
}
```
into snapshot.go